apps/src/componentLibrary/typography/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import classnames from 'classnames';
+
+const styles = require('./typography.module.scss').default;
+
+type SemanticTag =
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'h6'
+  | 'p'
+  | 'strong'
+  | 'em'
+  | 'figcaption';
+
+type VisualApproach =
```

Comment:
Maybe `Presentation` or `ViewStyle` or something? `VisualApproach` sounds a little unclear to me, but happy to discuss!

Reply:
"Visual Approach" is a UI term, but it might be helpful to give a definition here for the sake of people unfamiliar with it.

Reply:
@sanchitmalhotra126 @cat5inthecradle agree, `visualApproach` is not the most self-documenting name 😄 .
How about `visualAppearance` or `visualRepresentation` ?